dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/config/SpringConfigWeb.java

```diff
@@ -120,12 +116,6 @@ public FilterRegistrationBean logSessionIdFilter() {
     return registration;
   }
 
-  @Bean
-  public LanguageSortingHelper languageSortingHelper(
```

Comment:
Wird der wirklich nicht mehr benötigt? In fast allen Admin-Controllern wird im Konstruktor der `LanguageSortingHelper` noch verwendet.

Reply:
Paßt :+1: